src/lib/viewers/controls/zoom/ZoomControls.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import isFinite from 'lodash/isFinite';
+import IconZoomIn10 from '../icons/IconZoomIn10';
+import IconZoomOut10 from '../icons/IconZoomOut10';
+import './ZoomControls.scss';
+
+export type Props = {
+    maxScale?: number;
+    minScale?: number;
+    onZoomIn: () => void;
+    onZoomOut: () => void;
+    scale?: number;
+};
+
+export const MAX_SCALE = 100;
```

Comment:
Should this be `1`?

Reply:
I don't think so? A scale of `1` would equate to a zoom `100%`, which would be a low maximum.